Cilsil/Sil/TypeEnvironment.cs

```diff
@@ -74,8 +75,21 @@ public TypeEnvironment()
         /// <param name="path">The file path.</param>
         public void WriteToFile(string path)
         {
+            var ascii = Encoding.GetEncoding(
+                "us-ascii",
+                new EncoderReplacementFallback(string.Empty),
+                new DecoderExceptionFallback()
+            );
+
             var serializer = JsonSerializer.Create(JsonSerializerSettings);
-            using (var streamWriter = new StreamWriter(path ?? "./tenv.json"))
+
+            var output = path ?? "./cfg.json";
```

Comment:
It seems that this `WriteToFile` function is for type environment instead of CFG.

Reply:
Yes....we cannot crash the translation if there are types that fail to encode...

Reply:
ahhh good catch on cfg.json vs tenv.json :)